transformers4rec/utils/data_utils.py

```diff
@@ -0,0 +1,384 @@
+# type: ignore
```

Comment:
We should try to follow up pretty quickly with deprecating `preprocessing.py` so that we aren't stuck in a confusing state with two versions of this code at the same time. 

If it's not a ton of work (seems like it wouldn't be?) maybe tack that transition onto this PR?

Reply:
That's a good point. [This](https://github.com/NVIDIA-Merlin/Transformers4Rec/pull/619/commits/a2508999e5e8fa6ebc14b5d4f55da28e833c0504) commit does this.